src/test/reporters/dot.ts

```diff
@@ -21,7 +21,7 @@ import { FullResult, TestCase, TestResult } from '../../../types/testReporter';
 class DotReporter extends BaseReporter {
   private _counter = 0;
 
-  onTestEnd(test: TestCase, result: TestResult) {
+  override onTestEnd(test: TestCase, result: TestResult) {
```

Comment:
maybe we should say more "implements" instead of "extends"? (other reporters too)